systemtest/playlist.xml

```diff
@@ -18,12 +18,12 @@
 			<subset>SE90</subset>
 			<subset>SE100</subset>
 		</subsets>
-		<levels>
-			<level>extended</level>
-		</levels>
 		<groups>
 			<group>system</group>
 		</groups>
+		<levels>
+			<level>extended</level>
+		</levels>
```

Comment:
This change I don't think is necessary. Same for the following similar changes.

Reply:
Correct. It was a mistake. Fixed!